### PR TITLE
chore: build restful control plane in selfci image builder

### PR DESCRIPTION
**Pull request by rohan2794**
- build mayastor and restful control plane in selfci image builder
- update tests in slefci test profile

JIRA: https://mayadata.atlassian.net/browse/MQ-2495

### CODE REVIEW

File: scripts/test_lists.sh [hunk 1]
```diff
@@ -69,22 +69,17 @@ resource_check
 profiles[self_ci]="
 basic_volume_io
 io_soak
-maximum_vols_io
```
Comment (blaisedias) [inline scripts/test_lists.sh:72]: I think we should have comment(s) listing the tests removed, on the grounds that we would like to add them back in the future, and having a comment makes it easier to do that..

**Review by blaisedias — APPROVED** [review]
See minor comment.

**Review by blaisedias — APPROVED** [review]

**Review by payes — APPROVED** [review]

File: pipelines/Mayastor-e2e-selfci-image-builder/Jenkinsfile [hunk 2]
```diff
@@ -35,7 +35,12 @@ pipeline {
             script {
               common = load "./pipelines/common/common.groovy"
               test_tag = common.GetTestTag()
```
Comment (chriswldenyer) [inline pipelines/Mayastor-e2e-selfci-image-builder/Jenkinsfile:37]: I think we should have: def test_tag = ...

**Review by chriswldenyer — CHANGES_REQUESTED** [review]
We should use def for variables, to limit the scope.

File: pipelines/Mayastor-e2e-selfci-image-builder/Jenkinsfile (hunk 2 above)
Comment (rohan2794) [reply pipelines/Mayastor-e2e-selfci-image-builder/Jenkinsfile]: Done

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Pull request merged**